layouts/page/brand.html

```diff
@@ -0,0 +1,84 @@
+{{ define "hero" }}
+    <header class="header-hero header-hero-glow-bottom pt-12 pb-8">
+        <div class="header-hero-items container mx-auto">
+            <div class="header-hero-item text-center w-full py-8">
+                <h1 class="m-0">
+                    Pulumi Brand Resources
+                </h1>
+                <p>
+                    Downloadable logos and official brand colors.
+                </p>
+            </div>
+        </div>
+    </header>
+{{ end }}
+
+{{ define "main" }}
+    <nav class="nav-section-sticky">
+        <div class="container mx-auto">
+            <ul>
+                {{ range .Params.sections }}
+                    <li><a href="#{{ .anchor }}" class="uppercase">{{ .label }}</a></li>
+                {{ end }}
+            </ul>
+        </div>
+    </nav>
+
+    <section id="logo" class="mt-8 px-4 md:px-0">
```

Comment:
I think this needs to be `id="logos"`. The anchor tag in the front matter has a value of `logos` so the menu is not currently working.